cartographer/io/hybrid_grid_points_processor.cc

```diff
@@ -0,0 +1,85 @@
+#include "cartographer/io/hybrid_grid_points_processor.h"
+
+#include <memory>
+#include <string>
+#include <unordered_set>
+
+#include "Eigen/Core"
+#include "cartographer/common/make_unique.h"
+#include "cartographer/io/file_writer.h"
+#include "cartographer/io/points_batch.h"
+#include "cartographer/io/points_processor.h"
+#include "cartographer/mapping_3d/hybrid_grid.h"
+#include "cartographer/mapping_3d/range_data_inserter.h"
+#include "cartographer/sensor/range_data.h"
+#include "glog/logging.h"
+
+namespace cartographer {
+namespace io {
+
+namespace {
+
+std::unique_ptr<sensor::RangeData> RangeDataFromPointsBatch(
+    const PointsBatch& points_batch) {
+  auto range_data = common::make_unique<sensor::RangeData>();
+  range_data->origin = points_batch.origin;
+  for (const Eigen::Vector3f& point : points_batch.points) {
```

Comment:
The function is then two lines (one if you use {} initializer), how about you inline it?